10/onbuild/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:10.0.0
```

Comment:
should we drop the onbuilds now?

Reply:
https://github.com/docker-library/official-images/issues/2076

Reply:
Should we remove them all together or just for v10.x?

Reply:
Just for 10 I think, to not break people depending on them for older versions of node

Reply:
Updated!